downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc

```diff
@@ -38,9 +38,13 @@ The automation hub UI is available by default at:
 https://<hub node>:444
 ----
 
-.Accessing Automation EDA UI
+Login as the admin user with the password you created for *controller_admin_password*.
+
+.Accessing Automation {EDAName} UI
 
-The Automation EDA UI is available by default at:
+The Automation {EDAName} UI is available by default at:
```

Comment:
Same here with regard to "Automation". It should be removed of lowercase since it's not part of the official product name.